32-bit.md

```diff
@@ -0,0 +1,25 @@
+# 32-bit support of Git for Windows
+
+While Git for Windows v1.x was only ever offered as 32-bit installer (i.e. targeting the i686 CPU architecture), as of Git for Windows v2.x (which happened mid-2015), there have been two variants: the 32-bit and the 64-bit (x86\_64) one.
+
+However, due to Git for Windows' reliance on the MSYS2 runtime, which dropped 32-bit support as of version 3.4.x, Git for Windows will start to phase out 32-bit support after Git for Windows v2.40.x.
```

Comment:
Maybe a link to the MSYS runtime 3.4.x release notes/docs about dropping 32-bit support? For those that care to dive deeper?

Reply:
Done! I also reworded the introduction in places where I found it klunky.